data/CO/incentives.json

```diff
@@ -2083,7 +2078,7 @@
       "rebate"
     ],
     "item": "heat_pump_clothes_dryer",
-    "program": "co_empireElectricAssociationAndTri-StateGenerationAndTransmissionAssociation,InceNERGYEFFICIENCYPRODUCTSPROGRAM",
+    "program": "co_empireElectricAssociationAndTri-StateGenerationAndTransmissionAssociation,Inc_eNERGYEFFICIENCYPRODUCTSPROGRAM",
```

Comment:
I don't think we want to allow for custom program names because it'll make it harder for the script to scale. To prevent duplicates, I'm just concatenating state, authority, and program, but that does mean you occasionally get really long ones.

These are entirely internal, so I think we should keep for now and revisit if it ends up being a recurring problem.